.github/workflows/version.yml

```diff
@@ -0,0 +1,73 @@
+# to test locally please add push to the on: workflow selection, normally we will only want to
+# test this nightly so on push isn't needed for that but is for using act locally to later run and check.
```

Comment:
```suggestion
# To test locally with `act` (https://github.com/nektos/act), run `act -W .github/workflows/version.yml --detect-event`
```

no need to add the `push` trigger to test! you can use the above command to test locally without any changes